File: POLSKI_SPOJ/C/PRZEDSZK.c
#include <stdio.h>

void lcmOfTwoNumbers(int num1, int num2){
    int temp;
    int mem1 = num1;
    int mem2 = num2;
    while (num2 != 0) {
        temp = num2;
        num2 = num1 % num2;
        num1 = temp;
    }
    int gcd = num1;
    int lcm = (mem1 * mem2) / gcd;
    printf("%d\n", lcm);
}

int main() {
    int numberOfCases;
    scanf("%d", &numberOfCases);
    if(numberOfCases >= 1 && numberOfCases <= 20){
        for(int i = 0; i < numberOfCases; i++){
            int ammount1;
            int ammount2;
            scanf("%d %d", &ammount1, &ammount2);
            lcmOfTwoNumbers(ammount1, ammount2);
        }
    }
    return 0;
}

File: POLSKI_SPOJ/Python/PRZEDSZK.py
def nwd(a, b):
    while b:
        a, b = b, a % b
    return a


number_of_cases = input()
for x in range(int(number_of_cases)):
    line = input()
    numbers = list(line.split(" "))
    num1 = int(numbers[0])
    num2 = int(numbers[1])
    print(int((num1 * num2) / nwd(num1, num2)))
